src/plugins/data/public/interpreter/util/render_function.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
why is this in util and not in the registries folder ?

Reply:
Refactored.